components/invitation_modal/invitation_modal_members_step.jsx

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {isEmail} from 'mattermost-redux/utils/helpers';
+
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+import InviteIcon from 'components/svg/invite_icon';
+import UsersEmailsInput from 'components/widgets/inputs/users_emails_input.jsx';
+
+import BackIcon from 'components/svg/back_icon';
+
+import {getSiteURL} from 'utils/url.jsx';
+import debouncePromise from 'utils/debounce_promise.jsx';
+import {t} from 'utils/i18n.jsx';
+
+import './invitation_modal_members_step.scss';
+
+export default class InvitationModalMembersStep extends React.Component {
+    static propTypes = {
+        inviteId: PropTypes.string.isRequired,
+        goBack: PropTypes.func,
+        searchProfiles: PropTypes.func.isRequired,
+        onEdit: PropTypes.func.isRequired,
+        onSubmit: PropTypes.func.isRequired,
+    }
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            usersAndEmails: [],
+        };
+    }
+
+    usersLoader = async (term) => {
+        if (isEmail(term)) {
+            return [];
+        }
+        const {data} = await this.props.searchProfiles(term);
```

Comment:
Does this need to handle an error response?

Reply:
Here I have changed the code to not return results on fail.